File: src/main/java/com/manson/domain/itemextractor/CharacterInventory.java
package com.manson.domain.itemextractor;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.manson.domain.AbstractObject;
import com.manson.domain.fo76.AccountInfoData;
import com.manson.domain.fo76.CharacterInfoData;
import com.manson.domain.fo76.ItemDescriptor;
import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = false)
@ToString(callSuper = true)
public class CharacterInventory extends AbstractObject {

    private List<ItemDescriptor> playerInventory;
    private List<ItemDescriptor> stashInventory;
    @JsonProperty("AccountInfoData")
    private AccountInfoData accountInfoData;
    @JsonProperty("CharacterInfoData")
    private CharacterInfoData characterInfoData;

}

File: src/main/kotlin/com/manson/domain/itemextractor/CharacterInventory.kt
package com.manson.domain.itemextractor

import com.fasterxml.jackson.annotation.JsonProperty
import com.manson.AbstractObject
import com.manson.domain.fo76.AccountInfoData
import com.manson.domain.fo76.CharacterInfoData
import org.apache.commons.lang3.builder.ToStringBuilder
import org.apache.commons.lang3.builder.ToStringStyle

open class CharacterInventory : AbstractObject() {
    var playerInventory: MutableList<ItemDescriptor> = ArrayList()
    var stashInventory: MutableList<ItemDescriptor> = ArrayList()

    @JsonProperty("AccountInfoData")
    var accountInfoData: AccountInfoData = AccountInfoData()

    @JsonProperty("CharacterInfoData")
    var characterInfoData: CharacterInfoData = CharacterInfoData()
    override fun toString(): String {
        return ToStringBuilder(this, ToStringStyle.NO_CLASS_NAME_STYLE)
            .append("playerInventory", playerInventory)
            .append("stashInventory", stashInventory)
            .append("accountInfoData", accountInfoData)
            .append("characterInfoData", characterInfoData)
            .toString()
    }
}